application/src/main/resources/ecc.yml

```diff
@@ -20,6 +20,24 @@
 ##
 connection:
   cql:
+    ## Configuration to define datacenters and hosts for ecchronos
+    ## to connect to. The 'enabled' property, set to false by default,
+    ## indicates whether the "datacenter aware" functionality will be active.
+    ## When 'enabled' is true, the application will use the configurations
+    ## specified in 'datacenters', connecting to the listed hosts.
+    datacenterAware:
+      enabled: false
+      datacenters:
```

Comment:
When config is disabled, should it be commented out or just removed, perhaps?

Reply:
Done